src/vs/editor/contrib/codeAction/browser/codeActionMenu.ts

```diff
@@ -47,6 +48,7 @@ function stripNewlines(str: string): string {
 
 export interface CodeActionShowOptions {
 	readonly includeDisabledActions: boolean;
```

Comment:
Maybe make `fromLightBulb` optional here so normal callers don't have to care about it (just the lightbulb)